### PR TITLE
bump-formula-pr: add --fork-org flag

### PR DESCRIPTION
**Pull request by dawidd6**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

Closes: https://github.com/Homebrew/brew/issues/11143

Tested here: https://github.com/dawidd6/homebrew-test/pull/396

### CODE REVIEW

**Comment by brendongo** [comment]
@dawidd6 do you not need to modify check_fork_exists to also take an org? I think the URL it sends to right now assumes it is a personal fork

```
  def check_fork_exists(repo, organization = nil)
    _, reponame = repo.split("/")

    # If organization name is not given, get username
    owner = organization || API.open_rest(url_to("user")) { |json| json["login"] }
    json = API.open_rest(url_to("repos", owner, reponame))

    return false if json["message"] == "Not Found"

    true
  end
```

**Comment by dawidd6** [comment]
Right, good catch. Pushed fix.

File: Library/Homebrew/utils/github.rb [hunk 1]
```diff
@@ -487,7 +488,10 @@ def create_bump_pr(info, args:)
       changed_files += additional_files if additional_files.present?
 
       if args.dry_run? || (args.write? && !args.commit?)
-        ohai "try to fork repository with GitHub API" unless args.no_fork?
+        unless args.no_fork?
+          ohai "try to fork repository with GitHub API"
+          ohai "..into `#{args.fork_org}` GitHub organization" if args.fork_org
```
Comment (MikeMcQuaid) [inline Library/Homebrew/utils/github.rb:493]:
```suggestion
          puts "into `#{args.fork_org}` GitHub organization" if args.fork_org
```

or something as the `ohai ".."` is a bit weird.

**Review by MikeMcQuaid — COMMENTED** [review]
Looks good! One mild comment.

File: Library/Homebrew/utils/github.rb [hunk 2]
```diff
@@ -487,7 +488,10 @@ def create_bump_pr(info, args:)
       changed_files += additional_files if additional_files.present?
 
       if args.dry_run? || (args.write? && !args.commit?)
-        ohai "try to fork repository with GitHub API" unless args.no_fork?
+        unless args.no_fork?
+          ohai "try to fork repository with GitHub API"
+          puts "..into `#{args.fork_org}` GitHub organization" if args.fork_org
```
Comment (nandahkrishna) [inline Library/Homebrew/utils/github.rb:493]: This still looks a bit weird (all lines are `ohai`s except this one, and the `..`):
```
...
==> brew audit --formula clair.rb
==> try to fork repository with GitHub API
..into `Blah` GitHub organization
==> git add /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/clair.rb
...
```

Would something this be okay?:
```suggestion
          fork_message = "try to fork repository with GitHub API#{" into `#{args.fork_org}` organization"}"
          ohai fork_message
```

Comment (dawidd6) [reply Library/Homebrew/utils/github.rb:493]: Done

**Review by nandahkrishna — APPROVED** [review]

**Pull request merged**